webpack.config.js

```diff
@@ -208,7 +213,7 @@ const webpackConfig = ( mode ) => {
 					flagsConfig,
 					{
 						modes: [ 'development', 'production' ],
-						mode,
+						flagMode, // Default: mode; override with --flagMode={mode}
```

Comment:
This is particularly useful for E2E development as a way to use a development build with `watch` for quick iteration, without causing tests to fail due to extra development-only features.